handler.py

```diff
@@ -43,6 +43,7 @@ def main(event, _):
     # BEFORE setting up AWS resources
     verify_query = is_malformed_query(request_body, request_content_type)
```

Comment:
should we calculate clean type here? and pass it to `is_malformed_query` and then we can also use it on 58?

Reply:
Good suggestion. Fixed.